app/bundles/LeadBundle/Translations/en_US/messages.ini

```
@@ -673,6 +673,7 @@ mautic.company.company="Company"
 mautic.lead.company.viewleads_count="view contacts %count%"
 mautic.lead.lead.companies="Companies"
 mautic.lead.lead.segment.add.help="If you upload a CSV contact file to a segment, it will become a static list. Applying additional filters to this segment will not remove any contacts from that uploaded list."
+mautic.lead.lead.segment.filter.info="Any applicable contacts that match the filters will be added and contacts that no longer match will be removed. Contacts that were manually added will remain part of the list regardless of filters."
```

Tiny grammatical suggestion:

```suggestion
mautic.lead.lead.segment.filter.info="Any applicable contacts that match the filters will be added, and contacts that no longer match will be removed. Contacts that were manually added will remain part of the list regardless of filters."
```

---

Could we link out to the documentation, or would that be overkill?

---

@RCheesley thanks for the suggestion. I've just committed this change. 

I'm not sure about additional link. I haven't seen such a link anywhere in Mautic before. @npracht what do you think about this?

---

I think this is not the right approach adding a link hardly coded. Maybe we could think about a user journey plugin someday that people can install to have contextual help ?